apm-agent-plugins/apm-es-restclient-plugin/apm-es-restclient-plugin-6_4/src/test/java/co/elastic/apm/es/restclient/v6_4/ElasticsearchRestClientInstrumentationIT.java

```diff
@@ -83,24 +85,17 @@
     private static final String BAR = "bar";
     private static final String BAZ = "baz";
 
-    /**
-     * This Integration testing relies on <a href="https://github.com/dadoonet/testcontainers-java-module-elasticsearch">this
-     * ES testcontainer module</a> (will be replaced with an official version once merged into the
-     * <a href="https://github.com/testcontainers/testcontainers-java">testcontainers repo</a>)
-     */
     @BeforeClass
     public static void startElasticsearchContainerAndClient() throws IOException {
         // Start the container
-        // TODO: the env setting is to pass the bootstrap check of "vm.max_map_count" which fails on the CI env.
-        // TODO: Can be removed after changing to the official testcontainer
-        container = (ElasticsearchContainer) new ElasticsearchContainer().withEnv("discovery.type", "single-node");
+        container = new ElasticsearchContainer();
         container.start();
 
         // Create the client
         CredentialsProvider credentialsProvider = new BasicCredentialsProvider();
         credentialsProvider.setCredentials(AuthScope.ANY, new UsernamePasswordCredentials(USER_NAME, PASSWORD));
 
-        RestClientBuilder builder =  RestClient.builder(container.getHost())
+        RestClientBuilder builder =  RestClient.builder(new HttpHost(container.getHost().getHostName(), container.getHost().getPort()))
```

Comment:
if you update to 1.10.1 hotfix release, there will be no `getHost`, sorry for that.
But instead, we now provide `String getHttpHostAddress()`, so that you can use it like this:
```java
RestClient.builder(HttpHost.create(container.getHttpHostAddress()))
```